hack/verify-test-owners.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
@ixdy the main reqadon was, when i looked at the directory, it looked obvious that there were various update-*.sh scripts that updated different things and there was this solitary update_owners.py which did not appear to me as one of them. 

Also, since the python script runs go commands, i thought adding KUBE_ROOT would be prudent as well just like the other scripts. hence the wrapper script.